testcafe/runner-drive.js

```diff
@@ -4,8 +4,8 @@ async function runRunner() {
   const tc = await createTestCafe()
   const runner = await tc.createRunner()
   const response = await runner
-    .src(['testcafe/tests/drive_nav.js'])
-    .browsers(['firefox:headless'])
+    .src(['testcafe/tests/drive_nav.js', 'testcafe/tests/drive_sharing.js'])
+    .browsers(['chrome --start-maximized'])
```

Comment:
Does this mean we need to install chrome instead of firefox on travis?

Reply:
Since Chrome is the most used browsers, we want to test on it first. So yes ?